int64 start
string pointCloudFilePath
---
bool success
